src/Datadog.Trace/SpanStatus.cs

```diff
@@ -0,0 +1,119 @@
+namespace Datadog.Trace
```

Comment:
Taken from https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/src/OpenTelemetry.Api/Trace/Status.cs

Reply:
maybe I should mention it in comment?

Reply:
I like the idea of having that mentioned in a comment.

Reply:
done